tests/vmi_multus_test.go

```diff
@@ -871,14 +879,76 @@ var _ = Describe("SRIOV", func() {
 	})
 })
 
+var _ = Describe("Macvtap", func() {
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+	macvtapNetworkName := "net1"
+
+	createCirrosVMIWithMacvtapStaticIP := func(virtClient kubecli.KubevirtClient, networkName string, ifaceName string, ipCIDR string, mac *string) *v1.VirtualMachineInstance {
+		vmi := tests.NewRandomVMIWithMacvtapInterfaceEphemeralDiskAndUserdata(tests.ContainerDiskFor(tests.ContainerDiskCirros), "#!/bin/bash\necho 'hello'\n", networkName)
+		if mac != nil {
+			vmi.Spec.Domain.Devices.Interfaces[0].MacAddress = *mac
+		}
+		_, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+		Expect(err).ToNot(HaveOccurred())
+
+		// configure the client VMI
+		tests.WaitUntilVMIReady(vmi, tests.LoggedInCirrosExpecter)
+		configInterface(vmi, ifaceName, ipCIDR, "\\$ ", true)
+		return vmi
+	}
+
+	tests.BeforeAll(func() {
```

Comment:
What about BeforeTestCleanup? Whatever that does https://github.com/kubevirt/kubevirt/blob/bd347183ac5a440b2101069bd56ffe1eb299d5e9/tests/vmi_multus_test.go#L590